pkgs/development/tools/github-commenter/default.nix

```diff
@@ -1,16 +1,18 @@
-{ lib, fetchFromGitHub, buildGoPackage }:
+{ lib, fetchFromGitHub, buildGoModule }:
 
-buildGoPackage rec {
+buildGoModule rec {
   pname = "github-commenter";
-  version = "0.5.0";
+  version = "0.8.0";
 
   src = fetchFromGitHub {
     owner = "cloudposse";
     repo = pname;
     rev = version;
-    sha256 = "0y7yw7x8gqfbkqdfrwd9lffx3rrp62nz1aa86liy2dja97dacpij";
+    sha256 = "HgiCgyig+49g275G6zZ0kGTxt1TSfFK8kt+SOf4ei74=";
   };
 
+  vendorSha256 = "Gw+cR5sA5MGuclcvur8olmRtK04LDP5vKJ5k7yZO3B0=";
+
   goPackagePath = "github.com/cloudposse/${pname}";
 
   goDeps = ./deps.nix;
```

Comment:
You can drop this line and remove the file `deps.nix` - these are not needed when using Go modules

Reply:
ah, indeed. done